content/building/build-versioning.md

```diff
@@ -46,3 +46,112 @@ set -x
+{{</notebox>}}
+
+Alternatively, if you use `YAML` configuration, you may just export the value to an environment variable and use it under your `CFBundleVersion` in `Info.plist`.
+
+## Use Codemagic CLI Tools to get the latest build number from Google Play
+
+Use [get-latest-build-number]() action from [google-play]() Codemagic CLI Tool to get the latest build number from Google Play.
+
+In order to do that, you need to provide API access to Google Play Developer API by providing `GCLOUD_SERVICE_ACCOUNT_CREDENTIALS` as arguments to the action, similarly how it's done while for [Google Play publishing](https://docs.codemagic.io/publishing-yaml/distribution/#google-play).
+
+Package name of the app in Google Play Console (Ex: com.google.example)
+
+
+Additionally, you will need to provide the package name of the app in Google Play Console (Ex. `com.google.example`).
+
+### Creating Google service account credentials
+
+You will need to set up a service account in Google Play Console and create a JSON key with credentials. See how to [set up a service account and create a key](../knowledge-base/google-play-api/#setting-up-the-service-account-on-google-play-and-google-cloud-platform).
+
+Save the API key as an [environment](../getting-started/yaml#environment) variable. Make sure to [encrypt](./encrypting/#encrypting-sensitive-data) the values of the variable before adding it to the configuration file.
+
+```yaml
+environment:
+  vars:
+    GCLOUD_SERVICE_ACCOUNT_CREDENTIALS: Encrypted(...)
+```
+
+{{<notebox>}}
+Alternatively, credentials can be specified as a command argument with the dedicated flag, see the details [here](). But anyway you should have them in environment variables so that they can be decrypted. In that case, the environment variable will be fallback for missing value in the script.
+{{</notebox>}}
+
+Once you have the Google Play Developer API access set with the mentioned above environment variable, you can get the build number using the tool
+
+```bash
+export GCLOUD_SERVICE_ACCOUNT_CREDENTIALS=$(echo $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS | base64 --decode) # if you encrypted the file itself, not its content
+LATEST_BUILD_NUMBER=google-play get-latest-build-number --package-name 'com.google.example' # use your own package name
+```
+
+{{<notebox>}}
+By default, the action will try to get the latest build number as the maximum build number across all tracks (`internal`, `alpha`, `beta`, `production`). If you want to limit the search, you can specify a particular track(s) with `--tracks` optional argument. Check the details [here]().
```

Comment:
`here` is a broken link, I think you meant to link to CLI tools? I would say `with optional argument '--tracks' defined in the [Codemagic CLI tool](... link to cli tools arguments for this command)`.

Reply:
fixed